README.md

```diff
@@ -396,18 +397,56 @@ authClient.signIn({
 
 ### `signOut()`
 
-Signs the user out of their current Okta [session](https://developer.okta.com/docs/api/resources/sessions#example).
+Signs the user out of their current Okta [session](https://developer.okta.com/docs/api/resources/sessions#example) and clears all tokens stored locally in the `TokenManager`. The Okta session can be closed using either an `XHR` method or a `redirect` method. By default, the `XHR` method is used. Here are some points to consider when deciding which method is most appropriate for your application:
```

Comment:
I think it's better to link to the root of that docs page, as the top of that page has the high level information about what the session is.  

`Okta [session](https://developer.okta.com/docs/api/resources/sessions#example)` 
->  
`[Okta session](https://developer.okta.com/docs/api/resources/sessions)`